docs/queries/filters.md

```diff
@@ -39,54 +53,75 @@ Natural language notation:
 - `last friday`
 - `14 days ago`
 - `in two weeks`
+- `14 October` (the current year will be used)
+- `May` (1st May in the current year will be used)
 
 Note that if it is Wednesday and you write `tuesday`, Tasks assumes you mean "yesterday", as that is the closest Tuesday.
 Use `next tuesday` instead if you mean "next tuesday".
 
-When the day changes, relative dates like `due today` are re-evaluated so that the list stays up-to-date.
-
-### Date ranges
-
-`<date range>` can be given in 2 ways.
+### Searching date ranges
 
 {: .released }
 Date range searches were introduced in Tasks 1.26.0.
 
-#### Date ranges in formal notation
+Tasks allows date searches to specify a pair of dates, `<date range>` .
 
-`<date range>` may be specified as 2 valid dates in `YYYY-MM-DD` format. Dates on either end are included, that is it is an inclusive search.
+These searches are inclusive: the dates at either end are found by the search.
 
-`before <date range>` will match before the earliest date of the range. `after <date range>` will match after the latest date of the range.
+### Absolute date ranges
+
+`<date range>` may be specified as 2 valid dates in `YYYY-MM-DD` format.
+
+Dates on either end are included, that is, it is an inclusive search.
+
+- `before <date range>` will match before the earliest date of the range.
+- `after <date range>` will match after the latest date of the range.
 
 Notes:
 
-- If one of the `YYYY-MM-DD` dates is invalid, then it is ignored and the filter will behave as `<date>` not `<date range>`
-- Date range cannot be specified by 2 natural language dates eg `next monday three weeks`
-- `in` and `on` can be omitted
+- `in` and `on` may be omitted.
+- If one of the `YYYY-MM-DD` dates is invalid, then it is ignored and the filter will behave as `<date>` not `<date range>`.
+- Date range cannot be specified by 2 relative dates eg `next monday three weeks`.
+- We do not recommend this, but it is technically possible to specify absolute dates in words, such as `25th May 2023`.
+  - We have found that using two such dates can lead to ambiguity and unintended results when the [chrono](https://github.com/wanasit/chrono) library parses the dates in your filter.
 
-Example:
+Example absolute date ranges:
 
 - `2022-01-01 2023-02-01`
 
-#### Date ranges in natural language notation
+### Relative date ranges
+
+Tasks supports a very specific set of relative `<date range>` values: `last|this|next week|month|quarter|year`. The pipe (`|`) character means 'or'.
 
-Tasks supports natural language date ranges as: `last|this|next week|month|quarter|year`. Tasks will process this range and convert it to formal notation (`YYYY-MM-DD`) internally.
+Tasks will process these ranges, based on today's date, and convert them to absolute date ranges (`YYYY-MM-DD YYYY-MM-DD`) internally.
+
+Dates on either end are included, that is, it is an inclusive search.
```

Comment:
This is true for absolute and relative ranges, how about moving it to ### Searching date ranges?

Reply:
If I understand correctly, I put it in all 3 sections in 'date ranges' - given its importance.